Multisig/UI/Transactions/TransactionStatusView.swift

```diff
@@ -9,8 +9,10 @@
 import SwiftUI
 
 struct TransactionStatusView: View {
+
```

Comment:
funny that I usually try to remove spaces here 😆

Reply:
yes, I think we need auto-formatter or linter as a  GitHub action/bot, so  that we get such review automatically 😄